content/changelog/2025-10-31.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Snapshots in Beta, Postgres 18 extensions, AI-ready prompts, and more
+---
+
+## Snapshots now in Beta with automated scheduling
+
+The [Backup & Restore](/docs/guides/backup-restore) page in the Neon Console combines **instant point-in-time restore** with **snapshots** to help you protect your data and recover from accidental changes, data loss, or schema issues. While the page remains in Early Access, the **snapshots feature has moved to Beta** with new capabilities for automated snapshot management.
```

Comment:
>While the page remains in Early Access, the **snapshots feature has moved to Beta** with new capabilities for automated snapshot management.

we're renaming the toggle (I'll link PR later)
the message should be that snapshots are available to all users (according to their plans) rather than to early access ones only.

Reply:
Update coming! TY